extensions/amp-consent/0.1/consent-state-manager.js

```diff
@@ -64,8 +64,8 @@ export class ConsentStateManager {
     /** @private {?function()} */
     this.consentReadyResolver_ = null;
 
-    /** @private {?Object<string, PURPOSE_CONSENT_STATE>} */
-    this.purposeConsents_ = null;
+    /** @private {Object<string, PURPOSE_CONSENT_STATE>|undefined} */
+    this.purposeConsents_ = undefined;
```

Comment:
You can remove this assignment. If nothing is assigned, it is implied as `undefined`.

Reply:
@micajuine-ho might have made this change due to the type issue i mentioned [above.](https://github.com/ampproject/amphtml/pull/32721#discussion_r579347751)